dptb/utils/interpolate/poly_interp.py

```
@@ -0,0 +1,282 @@
+            Local x values [n_points].
+        y_local : torch.Tensor
+            Local y values [n_channels, n_points].
+        xq_val : float
+            Single query point value.
+
+        Returns
+        -------
+        torch.Tensor
+            Interpolated values [n_channels].
+        """
+        n_channels, nn = y_local.shape
```

Ruff (0.15.12)

[warning] Unpacked variable `n_channels` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)